GearSpawner/Patches.cs

```diff
@@ -1,34 +1,49 @@
 ﻿using HarmonyLib;
 using Il2Cpp;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
 
 namespace GearSpawner;
 
 internal static class Patches
 {
-	[HarmonyPatch(typeof(LootTable), nameof(LootTable.GetPrefab))]
-	internal static class LootTable_GetPrefab
+	// patch the look tables as they initialize
+	[HarmonyPatch(typeof(LootTableData), nameof(LootTableData.Initialize))]
+	internal static class LootTableData_Awake
 	{
-		private static void Prefix(LootTable __instance)
+		private static void Postfix(LootTableData __instance)
 		{
-			LootTableManager.ConfigureLootTable(__instance);
+			LootTableManager.ConfigureLootTableData(__instance);
 		}
 	}
 
-	[HarmonyPatch(typeof(LootTable), nameof(LootTable.GetRandomGearPrefab))]
-	internal static class LootTable_GetRandomGearPrefab
+	//patch the scenes for loose items as they load
+	/// <summary>
+	/// Other than GameManager.SetAudioModeForLoadedScene(), QualitySettingsManager.ApplyCurrentQualitySettings is the last method called within GameManager.Update() before save file saving and loading occur. They only get called after the loading panel has closed, and they each only get called once. If GameManager.SetAudioModeForLoadedScene() was not inlined, it would be used instead.
+	/// </summary>
+	[HarmonyPatch(typeof(QualitySettingsManager), nameof(QualitySettingsManager.ApplyCurrentQualitySettings))]
+	internal static class GameManager_ApplyCurrentQualitySettings
 	{
-		private static void Prefix(LootTable __instance)
+		private static void Prefix()
 		{
-			LootTableManager.ConfigureLootTable(__instance);
+			GearSpawnManager.PrepareScene();
 		}
 	}
 
-	[HarmonyPatch(typeof(GameManager), nameof(GameManager.SetAudioModeForLoadedScene))]//Exists
-	internal static class GameManager_SetAudioModeForLoadedScene
+	// this patch is required to allow non md5 asset reference names to be "valid"
```

Comment:
I want more context and a justification for why the patch is in GearSpawner instead of ModComponent.